xilem_web/src/attribute_value.rs

```diff
@@ -11,6 +11,7 @@ pub enum AttributeValue {
     True, // for the boolean true, this serializes to an empty string (e.g. for <input checked>)
     I32(i32),
     U32(u32),
+    Usize(usize),
```

Comment:
I don't think that change is worth its own PR, I might add more types in a different PR though, as I think all values that make sense here should be supported IMO.